data/mods/gen7/random-teams.ts

```diff
@@ -1036,7 +1036,7 @@ export class RandomGen7Teams extends RandomGen8Teams {
 			) ? 'Choice Scarf' : 'Choice Specs';
 		}
 		if (counter.get('Special') === 3 && moves.has('uturn')) return 'Choice Specs';
-		if (counter.get('Physical') === 4 && species.id !== 'jirachi' &&
+		if (counter.get('Physical') === 4 && species.id !== 'jirachi' && species.id !== 'spinda' &&
```

Comment:
```suggestion
		if (counter.get('Physical') === 4 && species.id !== 'jirachi' &&
```
not necessary since Staller = lefties is higher priority